src/plugins/wb-delete-pii/wb-delete-pii.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @title WET-BOEW wb-delete-pii
+ * @overview This plugin delete Personal Identifiable Information (PII) before form submit
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @polmih
+ **/
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-delete-pii",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+
+	init = function( event ) {
+		var elm = wb.init( event, componentName, selector );
+
+		if ( elm ) {
+			var $elm = $( elm );
+
+			elm.addEventListener( "submit", function() {
+
+				// identify form elements that was assigned to be scrubbed
+				$( "[data-wb-pii]", $elm ).each( function() {
+
+					var $elmInput = $( this ),
+						elmInputVal = $elmInput.val(),
+						inpValLenght = elmInputVal.length;
+
+					if ( $elmInput.attr( "data-wb-pii" ) === "true" ) {
```

Comment:
Is it necessary to set and check for a value that is equal to "true". What other purpose or other value can be set? Is it for a future integration with the form validation plugin.